src/Workspaces/VisualBasic/Portable/Formatting/Engine/Trivia/VisualBasicTriviaFormatter.vb

```diff
@@ -69,7 +69,20 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Formatting
 
             ' line continuation
             If trivia2.Kind = SyntaxKind.LineContinuationTrivia Then
-                Return LineColumnRule.ForceSpacesOrUseAbsoluteIndentation(spacesOrIndentation:=1)
+                If trivia1.Kind = SyntaxKind.None Then
+                    ' File or line or trivia starting with _ requires a leading space
+                    Return LineColumnRule.ForceSpacesOrUseAbsoluteIndentation(spacesOrIndentation:=1)
+                ElseIf trivia1.Kind = SyntaxKind.LineContinuationTrivia Then
+                    ' _ line followed by another _ and the _ should align
+                    Return LineColumnRule.ForceSpacesOrUseDefaultIndentation(spaces:=0)
+                ElseIf trivia1.Kind = SyntaxKind.CommentTrivia Then
+                    ' _ ' Comment aligns with the ' instead of the _ causing each line to move right
+                    ' 2 space
+                    Return LineColumnRule.ForceSpacesOrUseDefaultIndentation(spaces:=-1)
```

Comment:
so, in terms of a comment, this isn't really helpful.  What i'd like is both an example of waht the original VB code looks like that this case is trying to handle, what we want the final code to look like, and why this particular method/arguent is the right thing to do that.  i.e. why do we use `ForceSpacesOrUseDefaultIndentation`?  why do we pass `spaces:=-1` to it?  

I honestly don't know the answers, and reviewing the PR isn't informative enough.  As such, the code needs effective comments to make it clear.